src/static/helpers/slasHelper.ts

```diff
@@ -123,18 +124,20 @@ export async function authorize(
 
   // set manual redirect on server since node allows access to the location
   // header and it skips the extra call. In the browser, only the default
+  // set manual redirect for external idp login so shoppers can grab the authorization url
   // follow setting allows us to get the url.
   /* istanbul ignore next */
   slasClientCopy.clientConfig.fetchOptions = {
     ...slasClient.clientConfig.fetchOptions,
     redirect: isBrowser ? 'follow' : 'manual',
+    mode: 'no-cors',
```

Comment:
Is this intended? It is a concerning mode to have it set by default.

Reply:
Hey @alexvuong - I've been doing some testing with these helpers in the `template-retail-react-app` and found that the redirects don't work without this being set. Any ideas?

Reply:
Would it work if the loginIDPUser url has the `Access-Control-Allow-Origin` response header? It is usually the site we redirect to need to regconize the allowed origin and response header accordingly, then request would pass. 🤔